exercises/run-length-encoding/example.js

```diff
@@ -0,0 +1,13 @@
+
+export const encode = (plaintext) => {
+  const consecutiveChars = /([\w\s])\1*/g;
+  return plaintext.replace(consecutiveChars,
+      match => (match.length > 1 ? match.length + match[0] : match[0]));
+};
+
+
+export const decode = (cypher) => {
```

Comment:
`encodedText` makes sense for me. If you rename it, would you mind to rename `plaintext` to `plainText` on line 2?